src/main/java/com/alkemy/ong/service/impl/OrganizationServiceImp.java

```diff
@@ -0,0 +1,38 @@
+package com.alkemy.ong.service.impl;
+
+import com.alkemy.ong.dto.OrganizationResponseDto;
+import com.alkemy.ong.exception.NotFoundException;
+import com.alkemy.ong.mapper.OrganizationResponseMapper;
+import com.alkemy.ong.model.Organization;
+import com.alkemy.ong.repository.OrganizationsRepository;
+import com.alkemy.ong.service.OrganizationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class OrganizationServiceImp implements OrganizationService {
+
+    private final OrganizationsRepository organizationsRepository;
+    private final OrganizationResponseMapper organizationResponseMapper;
+
+    @Override
+    public OrganizationResponseDto getOrganization(Long id) {
+
+        Organization org = null;
+        Optional<Organization> result = organizationsRepository.findById(id);
```

Comment:
It would be better use a more functional style, something like that:

 ```java
   public OrganizationResponseDto getOrganization(Long id) {
        return organizationsRepository.findById(id)
                .map(organizationResponseMapper::toOrganizationResponseDto)
                .orElseThrow(() -> new NotFoundException("Organization id not found - " + id));
    }
```